sigmastate/src/main/scala/sigmastate/eval/CostingDataContext.scala

```diff
@@ -129,14 +144,10 @@ case class CAvlTree(treeData: AvlTreeData) extends AvlTree with WrapperOf[AvlTre
     this.copy(treeData = td)
   }
 
-  private def createVerifier(proof: Coll[Byte]): BatchAVLVerifier[Digest32, Blake2b256.type] = {
+  override def createVerifier(proof: Coll[Byte]): AvlTreeVerifier = {
```

Comment:
ScalaDoc missed

Reply:
This is override, which means look for ScalaDocs in the base trait, added it there.